src/Symfony/Bundle/FrameworkBundle/Command/SecretsRevealCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Secrets\AbstractVault;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @internal
+ */
+#[AsCommand(name: 'secrets:reveal', description: 'Reveal a secrets value')]
+final class SecretsRevealCommand extends Command
+{
+    public function __construct(
+        private readonly AbstractVault $vault,
+        private readonly ?AbstractVault $localVault = null,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('name', InputArgument::REQUIRED, 'The name of the secret to reveal', null, fn () => array_keys($this->vault->list()))
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command reveals all stored secrets.
```

Comment:
```suggestion
The <info>%command.name%</info> command reveals a stored secret.
```

Reply:
This is a great example why you shouldn't copy/paste your old code 😅

Fixed